catalogs/fermi/README.rst

```diff
@@ -1,10 +1,25 @@
 Fermi catalogs
 ==============
 
-This is a copy of the Fermi-LAT catalogs.
+This is a copy of the Fermi-LAT catalogs that are licenses under `CC0 <https://creativecommons.org/publicdomain/zero/1.0/>`_.
+The Fermi Data policy can be found on `this page <https://fermi.gsfc.nasa.gov/ssc/data/policy/>`_.
 
-We'd like to use those for tests and examples in Gammapy,
-and had lots of timeouts downloading those from the FSSC server.
+The available catalogues are:
+ - LAT Second Catalog of Gamma-ray Pulsars (2PC):
+    - Reference: `A. A. Abdo et al., 2013, ApJS 208 17 <https://iopscience.iop.org/article/10.1088/0067-0049/208/2/17>`_
+ - LAT Third Catalog of Gamma-ray Pulsars (3PC):
+    - Reference: `D. A. Smith et al., 2023, ApJ 958 191 <https://iopscience.iop.org/article/10.3847/1538-4357/acee67>`_
 
-So while this is non-ideal, we're bundling a copy af the
-FITS catalog files here to make this more reliable.
+The gammapy team have used the following Fermi catalogues to create Fermi-LAT sources catalogues
+after 8 , 12 and 14 years of observations:
+
+- LAT 8-year Source Catalog (4FGL):
+    - Reference: `S. Abdollahi et al., 2020, ApJS 247 33 <https://iopscience.iop.org/article/10.3847/1538-4365/ab6bcb>`_
+- LAT 12-year Source Catalog (4FGL-DR3):
+    - Reference: `S. Abdollahi et al., 2020, ApJS 247 33 <https://iopscience.iop.org/article/10.3847/1538-4365/ab6bcb>`_
```

Comment:
The 2020 reference should not appear here

Reply:
The Fermi people mention to me to refer to this paper because the same methodology has been used. But indeed, I can add this paper, as stated here (https://fermi.gsfc.nasa.gov/ssc/data/access/lat/12yr_catalog/)